### PR TITLE
adding support files

### PR DESCRIPTION
**Pull request by aaitor**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,8 +4,8 @@
 	<modelVersion>4.0.0</modelVersion>
 
 	<groupId>com.oceanprotocol.secretstore</groupId>
```
Comment (innoprenuer) [inline pom.xml]: should be (as per our maven group id) <groupId>com.oceanprotocol</groupId>

File: src/test/java/com/oceanprotocol/secretstore/auth/PublishConsumeIT.java [hunk 2]
```diff
@@ -1,49 +1,172 @@
...
+    private static TransactionManager txManager;
+    private static Credentials credentials;
+
+    private static final String TEST_PASSWORD= "";
+    private static final String TEST_FILE= "src/test/resources/8fff55a4-dbac-b059-6b8b-bfc7394ed98a.json.testaccount";
+
+    private static final String ACL_CONTRACT_ADDRESS= "0x6d6a34f2be1e76902a2fde049f317610cdf453eb";
+
+
+    private static final String ALICE_ADDRESS= "0x00a329c0648769A73afAc7F9381E08FB43dBEA72";
```
Comment (eruizgar91) [inline src/test/java/com/oceanprotocol/secretstore/auth/PublishConsumeIT.java]: Should we get this addresses from the ganache accounts?

File: src/test/java/com/oceanprotocol/secretstore/contracts/AccessServiceAgreementIT.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    private static AccessServiceAgreement serviceAgreement;
+    private static Web3j web3= null;
+    private static TransactionManager txManager;
+    private static Credentials credentials;
+    private static final String TEST_ADDRESS= "0x00a329c0648769A73afAc7F9381E08FB43dBEA72";
+    private static final String TEST_PASSWORD= "";
+    private static final String TEST_FILE= "src/test/resources/8fff55a4-dbac-b059-6b8b-bfc7394ed98a.json.testaccount";
+
+    private static final String CONTRACT_ADDRESS= "0x783f7d86c2634474d84636b56f5cc7dfd360ab15";
```
Comment (eruizgar91) [inline src/test/java/com/oceanprotocol/secretstore/contracts/AccessServiceAgreementIT.java]: The same about this addresses

File: src/test/java/com/oceanprotocol/secretstore/auth/PublishConsumeIT.java (hunk 2 above)
Comment (aaitor) [reply src/test/java/com/oceanprotocol/secretstore/auth/PublishConsumeIT.java]: user addresses are random addresses to give and revoke grants during the process, not the address of users in ganache. They don't need to have ETH so it's okay to have those random addresses.

File: src/test/java/com/oceanprotocol/secretstore/contracts/AccessServiceAgreementIT.java (hunk 3 above)
Comment (aaitor) [reply src/test/java/com/oceanprotocol/secretstore/contracts/AccessServiceAgreementIT.java]: Regarding the Contract address would be a good point to have this orchestrated. The idea is to use in the future typesafe config to orchestrate the integration tests in travis using Docker. But it's part of a bigger thing.

**Pull request merged**